Assets/HoloToolkit/Input/README.md

```diff
@@ -189,16 +190,30 @@ or in your Visual Studio Package.appxmanifest capabilities.
 **RecognizerStart** Set this to determine whether the keyword recognizer will start immediately or if it should wait for your code to tell it to start.
 
 #### Voice
-##### KeywordManager.cs
-Allows you to specify keywords and methods in the Unity Inspector, instead of registering them explicitly in code.  
-**IMPORTANT**: Please make sure to add the microphone capability in your app, in Unity under  
+
+**IMPORTANT**: Please make sure to add the Microphone and Music Library capabilities in your app, in Unity under  
```

Comment:
Why do we need the Music Library capability? I've never set that and haven't run into any issues with recognizers or microphone input.